src/ripple/ledger/View.h

```diff
@@ -56,24 +56,24 @@ enum FreezeHandling
     fhZERO_IF_FROZEN
 };
 
-bool
+[[nodiscard]] bool
 isGlobalFrozen (ReadView const& view,
     AccountID const& issuer);
 
-bool
+[[nodiscard] ]bool
```

Comment:
Nit: `[[nodiscard]] bool`, just for consistency

Reply:
Oooh!  Good eyes.  I'll go ahead and squash, but I'll fix that during the squash.

Thanks for all the help everybody!